modules/pensions/spec/sidekiq/pensions/pension_benefit_intake_job_spec.rb

```diff
@@ -0,0 +1,161 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'lighthouse/benefits_intake/service'
+require 'lighthouse/benefits_intake/metadata'
+
+RSpec.describe Pensions::PensionBenefitIntakeJob, :uploader_helpers do
+  stub_virus_scan
+  let(:job) { described_class.new }
+  let(:claim) { create(:pensions_module_pension_claim) }
+  let(:service) { double('service') }
+  let(:monitor) { double('monitor') }
+  let(:user_uuid) { 123 }
+
+  describe '#perform' do
+    let(:response) { double('response') }
+    let(:pdf_path) { 'random/path/to/pdf' }
+    let(:location) { 'test_location' }
+
+    before do
+      job.instance_variable_set(:@claim, claim)
+      allow(Pensions::SavedClaim).to receive(:find).and_return(claim)
+      allow(claim).to receive_messages(to_pdf: pdf_path, persistent_attachments: [])
+
+      job.instance_variable_set(:@intake_service, service)
+      allow(BenefitsIntake::Service).to receive(:new).and_return(service)
+      allow(service).to receive(:uuid)
+      allow(service).to receive(:request_upload)
+      allow(service).to receive_messages(location:, perform_upload: response)
+      allow(response).to receive(:success?).and_return true
+
+      job.instance_variable_set(:@pension_monitor, monitor)
+      allow(monitor).to receive :track_submission_begun
+      allow(monitor).to receive :track_submission_attempted
+      allow(monitor).to receive :track_submission_success
+      allow(monitor).to receive :track_submission_retry
```

Comment:
interesting that the linter did not complain about `receive_messages` being applied more